examples/ffs_1d.py

```diff
@@ -0,0 +1,33 @@
+import math
+import numpy as np
+from pyffs import ffs_sample, ffs
+from pyffs.func import dirichlet
+import matplotlib.pyplot as plt
+import os
+from util import plotting_setup
+
+fig_path = plotting_setup()
+
+T, T_c, N_FS = math.pi, math.e, 15
+N_samples = 512
+sample_points, _ = ffs_sample(T, N_FS, T_c, N_samples, mod=np)
+diric_samples = dirichlet(sample_points, T, T_c, N_FS)
+diric_FS = ffs(diric_samples, T, T_c, N_FS)[:N_FS]
+
+# plot time
+idx = np.argsort(sample_points)
+_, ax = plt.subplots(nrows=2, num="FS coefficients of Dirichlet", figsize=(10, 10))
```

Comment:
`fig, ax = plt.subplots(nrows=2, num="FS coefficients of Dirichlet", figsize=(10, 10))`